tools/build_defs/pkg/make_deb.py

```diff
@@ -281,7 +282,7 @@ def CreateChanges(output,
                      ), ('Checksums-Sha256', '\n' + ' '.join(
                          [checksums['sha256'], debsize, deb_basename]))])
   with open(output, 'w') as changes_fh:
-    changes_fh.write(changesdata)
+    changes_fh.write(changesdata.encode("utf-8"))
```

Comment:
If you would like to do future maintainers a favor, you could clean up the formatting on the block above this.
The current formatting is an unreadable mess. How about.

  changesdata = ''.join(
      MakeDebianControlField(*x)
      for x in [
          ('Format', '1.8'),
          ('Date', time.ctime(timestamp)),
          ('Source', package),
          ('Binary', package),
          ('Architecture', architecture),
          ('Version', version),
          ('Distribution', distribution),
          ('Urgency', urgency),
          ('Maintainer', maintainer),
          ('Changed-By', maintainer),
          ('Description', '\n%s - %s' % (package, short_description)),
          ('Changes', (
              '\n%s (%s) %s; urgency=%s'
              '\nChanges are tracked in revision control.') % (
                  package, version, distribution, urgency)),
          ('Files', '\n' + ' '.join(
              [checksums['md5'], debsize, section, priority, deb_basename])),
          ('Checksums-Sha1', '\n' + ' '.join(
              [checksums['sha1'], debsize, deb_basename])),
          ('Checksums-Sha256', '\n' + ' '.join(
              [checksums['sha256'], debsize, deb_basename]))])

Or even:
  changesdata = ''.join(
      MakeDebianControlField('Format', '1.8'),
      MakeDebianControlField('Date', time.ctime(timestamp)),
      ...

If not, I'll clean it up after.

Reply:
That's fair. I'm happy to do it. I was trying to keep my initial changes minimal so it was easier to review.

I like the below approach so will go ahead and do that unless you prefer the other method:
```python
changesdata = ''.join([
MakeDebianControlField(...),
MakeDebianControlField(...),
...
])
```

Reply:
That works for me too. It is easier to follow than "MakeDebianControlField(*x)"